src/plugins/controls/server/options_list/suggestion_queries/options_list_search_suggestions.ts

```diff
@@ -118,7 +118,7 @@ const suggestionAggSubtypes: { [key: string]: OptionsListSuggestionAggregationBu
 
       const suggestions = get(rawEsResult, `${basePath}.suggestions.buckets`)?.reduce(
         (acc: OptionsListSuggestions, suggestion: EsBucket) => {
-          acc.push({ value: suggestion.key, docCount: suggestion.doc_count });
+          acc.push({ value: String(suggestion.key), docCount: suggestion.doc_count });
```

Comment:
Can you add a test case src/plugins/controls/server/options_list/suggestion_queries/options_list_search_suggestions.test.ts with a response that contains numeric key values.

Reply:
Done in d44dc552edaebe827101feeafd146a5b3826761c

Reply:
Oops, realized there was some duplication happening - cleaned up in 4dc0515aac8927623b46c095fce6587832423e20